x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step/create_step.tsx

```diff
@@ -53,32 +79,114 @@ export const CreateStep: FC<Props> = ({ actions, state, step }) => {
+            <EuiFlexGroup direction="column" gutterSize="s">
+              <EuiFlexItem grow={false}>
+                <EuiFormRow
+                  helpText={i18n.translate(
+                    'xpack.ml.dataframe.analytics.create.startCheckboxHelpText',
+                    {
+                      defaultMessage:
+                        'If unselected, job can be started later by returning to the jobs list.',
+                    }
+                  )}
+                >
+                  <EuiCheckbox
+                    data-test-subj="mlAnalyticsCreateJobWizardStartJobCheckbox"
+                    id={'dataframe-create-start-checkbox'}
+                    label={i18n.translate(
+                      'xpack.ml.dataframe.analytics.create.wizardStartCheckbox',
+                      {
+                        defaultMessage: 'Start immediately',
+                      }
+                    )}
+                    checked={startChecked}
+                    onChange={(e) => {
+                      setStartChecked(e.target.checked);
+                      if (e.target.checked === false) {
+                        setFormState({ createIndexPattern: false });
+                      }
+                    }}
+                  />
+                </EuiFormRow>
+              </EuiFlexItem>
+              {startChecked ? (
+                <EuiFlexItem grow={false}>
+                  <EuiFormRow
+                    fullWidth
+                    isInvalid={
+                      (createIndexPattern && destinationIndexPatternTitleExists) ||
+                      createIndexPattern === false ||
+                      canCreateDataView === false
+                    }
+                    error={[
+                      ...(canCreateDataView === false
+                        ? [
+                            <EuiText size="xs" color="warning">
+                              {i18n.translate(
+                                'xpack.ml.dataframe.analytics.create.dataViewPermissionWarning',
+                                {
+                                  defaultMessage: 'You need permission to create data views.',
+                                }
+                              )}
+                            </EuiText>,
+                          ]
+                        : []),
+                      ...(createIndexPattern && destinationIndexPatternTitleExists
+                        ? [
+                            i18n.translate(
+                              'xpack.ml.dataframe.analytics.create.dataViewExistsError',
+                              {
+                                defaultMessage:
+                                  'A data view with the title {title} already exists.',
+                                values: { title: destinationIndex },
+                              }
+                            ),
+                          ]
+                        : []),
+                      ...(!createIndexPattern
```

Comment:
Would be good to hide this warning for the case where a data view already exists for the destination index, as in this case we know the warning doesn't apply. Can you just add an extra `!destinationIndexPatternTitleExists` here?

![image](https://user-images.githubusercontent.com/7405507/144625660-037cbaae-6cde-4230-81e2-981244491ff5.png)